INSTALL.md

```diff
@@ -2033,6 +2041,26 @@ around the problem by forcing the build procedure to use the following script:
 instead of the real clang. In which case it doesn't matter what clang version
 is used, as it is the version of the GNU assembler that will be checked.
 
+
+Notes on profile guided optimization
+------------------------------------
+
+Some compilers support the concept of profile guided optimization.  This feature
+allows a user to build openssl and use profiling data gathered while running an
+application such that it can then be rebuilt in a way that is optimized specifically
+for that application, increasing performance.  Currently this feature is built into
+the openssl build system for x86_64 only, but can be extended to other architectures.
+To take advantage of this feature.
```

Comment:
Sure it can be extended but that shouldn't be mentioned in the installation guide:
```suggestion
the openssl build system for x86_64 only.
```